core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java

```diff
@@ -559,7 +558,12 @@ private void addDatum(int internalId, String attrId, String attrVal, String attr
         // attribute value exists and if so, perfom an update
         if (attrType.equals(ClinicalAttribute.PATIENT_ATTRIBUTE)) {
             numPatientSpecificClinicalAttributesAdded++;
-            DaoClinicalData.addPatientDatum(internalId, attrId, attrVal.trim());
+            // add prefix for OS or DFS attributes
+            if (SurvivalAttributeUtil.isSurvivalAttribute(attrId)) {
```

Comment:
I suggest using the "has" approach as implemented in ImportClinicalData. MissingAttributeValues.  It makes for cleaner code - 

https://github.com/cBioPortal/cbioportal/blob/b69759630a61eb0040436987801ad57079cfdcfb/core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java#L81